deploy_aws.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script installs and configures the required command-line utilities, creates EC2 instances with the specified Amazon Machine Image (AMI) ID, ports, security group, and connects to the instance via Secure Shell (SSH). Additionally, it also stores the RSA key in AWS Secrets Manager for secure management.
+
+# Install aws cli
+if ( which aws > /dev/null ) 
+then
+  echo -e "\n==== Awscli currently installed ====\n"
+else 
+  echo -e "\n==== Installing awscli ====\n"
```

Comment:
We should break this up like we had done on the call with better assertions around the install.